retropie_packages.sh

```diff
@@ -73,5 +73,5 @@ fi
 printMsgs "console" "${__INFMSGS[@]}"
 
 if [[ -n $__joy2key_pid ]]; then
-    kill -INT $__joy2key_pid 2>/dev/null
+    kill -INT $__joy2key_pid 2>/dev/null || echo "Failed to kill joy2key"
```

Comment:
The problem I'm fixing with this change, is when joy2key isn't around, and kill tried to kill it. Kill fails with a non-zero exit code, and the fails.